lib/slack.py

```diff
@@ -25,8 +25,24 @@ def get_client(self):
         self._client = sc
         return self._client
 
+    def play_video(self, video_id):
+        # Plays a video in the current channel using the video id
+        video_url = "https://www.youtube.com/watch?v={}".format(video_id)
+        self.send_message(video_url)
+
+    def send_channel(self, channel_id, channel_name, thumbnail_url):
+        # Sends the youtube channel to the active user/slack channel
+        channel_url = "https://www.youtube.com/channel/{}".format(channel_id)
+
+        params = {'channel': 'tubeydev', 'text': "Here's the channel:", 'attachments':
+                  [{'fallback': channel_name + ' thumbnail', 'title': channel_name, "title_link": channel_url,
+                    "image_url": thumbnail_url}]}
+        client = self.get_client()
+        client.api_call("chat.postMessage", **params)
```

Comment:
We should make `send_message` take `params` as an optional argument so it can be used here. That way we are not duplicating code and it will be consistent with what should be called the `send_video` function